osu.Game.Rulesets.Mania/Beatmaps/ManiaBeatmapConverter.cs

```diff
@@ -71,6 +71,12 @@ public ManiaBeatmapConverter(IBeatmap beatmap, Ruleset ruleset)
             originalTargetColumns = TargetColumns;
         }
 
+        internal static int GetColumnCountForNonConvert(BeatmapInfo beatmap)
```

Comment:
```suggestion
        public static int GetColumnCountForNonConvert(BeatmapInfo beatmap)
```

Reply:
I intentionally made it `internal` just to limit scope, but I guess either is fine. Any rule of thumb to go by in these cases? Always public?

Reply:
Don't limit scope arbitrarily unless it's something that shouldn't be touched externally. If an external user touches this - who cares?

Reply:
I'm more of the thought that we should be using `internal` far more often to limit what could accidentally get used (then becoming a blocker for future changes due to breaking other systems) but that's a discussion for another day :).